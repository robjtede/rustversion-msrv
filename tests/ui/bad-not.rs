#[rustversion_msrv::any(not)]
struct S;

#[rustversion_msrv::any(not, not)]
struct S;

fn main() {}
