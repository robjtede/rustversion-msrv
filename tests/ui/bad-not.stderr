error: expected `(` after `not`
 --> tests/ui/bad-not.rs:1:25
  |
1 | #[rustversion_msrv::any(not)]
  |                         ^^^

error: expected `(`
 --> tests/ui/bad-not.rs:4:28
  |
4 | #[rustversion_msrv::any(not, not)]
  |                            ^
